packages/vscode-extension/src/handlers.ts

```diff
@@ -1396,7 +1396,7 @@ async function showLocalDebugMessage() {
   };
 
   ExtTelemetry.sendTelemetryEvent(TelemetryEvent.ShowLocalDebugNotification);
-  const appName = (await getAppName()) ?? "Teams App";
+  const appName = (await getAppName()) ?? "Your App";
```

Comment:
Could you help to add localization to this string?

Reply:
Sure